desktop/main.dev.ts

```diff
@@ -132,7 +133,11 @@ const createBrowserView = () => {
 };
 
 const addIpcEventListeners = () => {
-  ipcMain.handle(ipcConsts.GET_OS_THEME_COLOR, () => nativeTheme.shouldUseDarkColors);
+  ipcMain.handle(ipcConsts.GET_OS_THEME_COLOR, () => isDarkMode);
+
+  ipcMain.on(ipcConsts.SET_OS_THEME_COLOR, (_, { isDarkMode }) => {
```

Comment:
In case that tomorrow we also want to keep synced another prop - we have to duplicate this hunk of code? 🤔

Reply:
Additional props can be added inside the request